combirepo/commandline_parser.py

```diff
@@ -360,12 +360,20 @@ def __build_repository_pairs(self, arguments):
             repository_pairs.append(repository_pair)
         return repository_pairs
 
-    def __parse_packages_file(self, packages_file):
+    def __parse_packages_file(self, packages_file, disable_rpm_patching = False):
         packages_list = []
         if packages_file is not None:
             with open(packages_file, 'r') as pkg_file:
                 for package in pkg_file:
                     packages_list.append(package.strip())
+        packages_list.append("mic-bootstrap-x86-arm")
+        if not disable_rpm_patching:
+            deps = ["binutils", "bzip2", "cpio", "gcc", "glibc-devel",
```

Comment:
How did you create this list? Isn't there any automated method of building this list?

Reply:
I created it manually calling deps of rpm-build and its deps and added mic-bootstrap and qemu packages. At a stage of package list generation it is rather difficult to get deps without downloading packages.